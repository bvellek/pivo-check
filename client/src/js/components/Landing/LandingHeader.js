import React from 'react';
import LoginRegisterLink from './LoginRegisterLink';
import SVGLogo from '../SVGLogo';

const LandingHeader = () => (
  <header className="landing-header top-header" role="banner">
    <div className="landing-wrapper">
      <nav className="landing-nav" role="navigation">
        <LoginRegisterLink />
      </nav>
      <h1 className="landing-logo">
        <span className="visually-hidden">Pivo-Check</span>
        <SVGLogo />
      </h1>
    </div>
  </header>
);

export default LandingHeader;
