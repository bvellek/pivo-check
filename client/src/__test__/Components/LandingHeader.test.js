import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LandingHeader from '../../js/components/Landing/LandingHeader';

describe('<LandingHeader />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <LandingHeader />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
