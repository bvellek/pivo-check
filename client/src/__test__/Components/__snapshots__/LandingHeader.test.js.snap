// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingHeader /> it renders correctly 1`] = `
<header
  className="landing-header top-header"
  role="banner"
>
  <div
    className="landing-wrapper"
  >
    <nav
      className="landing-nav"
      role="navigation"
    >
      <LoginRegisterLink />
    </nav>
    <h1
      className="landing-logo"
    >
      <span
        className="visually-hidden"
      >
        Pivo-Check
      </span>
      <SVGLogo />
    </h1>
  </div>
</header>
`;
